From cfbd395ce973cb9b0dd8a4e2ddebdac6f07d42c8 Mon Sep 17 00:00:00 2001
From: Todd Whiteman <todd.whiteman@joyent.com>
Date: Wed, 13 Dec 2017 14:15:25 -0800
Subject: [PATCH] DOCKER-1050 add redirect handling for docker registry client
 getManifest() call Reviewed by: Trent Mick <trentm@gmail.com> Approved by:
 Trent Mick <trentm@gmail.com>

---
 CHANGES.md   | 4 ++++
 package.json | 4 ++--
 2 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/CHANGES.md b/CHANGES.md
index 30ce77d..55afdff 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,9 @@
 # IMGAPI changelog
 
+## 4.0.13
+
+- DOCKER-1095 add redirect handling for docker registry client getManifest
+
 ## 4.0.12
 
 - IMGAPI-651 for lx-dataset images `imgadm publish ...` inherits min_platform
diff --git a/package.json b/package.json
index 91d0fca..0b2fcd4 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "imgapi",
   "description": "Image API to manage images for SDC 7",
-  "version": "4.0.12",
+  "version": "4.0.13",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -11,7 +11,7 @@
     "bunyan": "1.8.8",
     "cmdln": "3.2.1",
     "dashdash": "1.10.0",
-    "docker-registry-client": "3.2.12",
+    "docker-registry-client": "3.2.13",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
     "expiring-lru-cache": "2.1.0",
     "extsprintf": "1.2.0",
-- 
2.21.0

